mteb/tasks/Clustering/fra/HALClusteringS2S.py

```diff
@@ -1,14 +1,18 @@
-from __future__ import annotations
+from collections import Counter
 
 import datasets
 import numpy as np
 
+from mteb.abstasks.AbsTaskClustering import AbsTaskClustering
+from mteb.abstasks.AbsTaskClusteringFast import AbsTaskClusteringFast
 from mteb.abstasks.TaskMetadata import TaskMetadata
 
-from ....abstasks.AbsTaskClustering import AbsTaskClustering
+NUM_SAMPLES = 2048
 
 
 class HALClusteringS2S(AbsTaskClustering):
+    superseeded_by = "HALClusteringS2SFast"
```

Comment:
Nice catch.
Sorry that i missed this the first time. We want to be consistent with the task name so this should be 
```suggestion
    superseeded_by = "HALClusteringS2S.v2"
```

The class name can remain the same.

Reply:
Ok, I'll change it here, but I already merged [pull request](https://github.com/embeddings-benchmark/mteb/pull/813/files) with name Fast